templates/journal/article.html

```diff
@@ -154,12 +156,12 @@ <h4>{% trans "Information" %}</h4>
                     {% endif %}
                 </ul>
                 {% for field in article.custom_fields %}
-                <h4>{% trans field.field.name %}</h4>
+                <h3>{% trans field.field.name %}</h3>
```

Comment:
These headings should be `<h2>` for WCAG compliance, otherwise they will be subheadings of the last `h2` in the article